### ISSUE TITLE
Strange 0101 class issue

### ISSUE THREAD

**Issue by jblacksberg** [Pattern-Labs/django-bom#4]
trying to add a 0101 class item to an existing 0101 class sub assembly seems to fail. No error message. Only failing with 0101 classes created today (I was able to add a prior 0101 class item without issue). Honestly weird and hard to describe/characterize. I have a workaround to create a dummy item with a different class then change the classes to match but that kinda sucks.

**Comment by jblacksberg** [comment]
Actually i figured something out. It IS adding these parts but they are not showing up in the "indented view" only in the flat view. I just had ait happen with a different new part in a different class that had no relation. So i guess I need to use the flat view for now? fuck

**Comment by jblacksberg** [comment]
![image](https://user-images.githubusercontent.com/105745183/218829252-bba8b4cc-2c56-4b4d-b1af-7a389c7a4bd3.png)

**Comment by jblacksberg** [comment]
![image](https://user-images.githubusercontent.com/105745183/218829365-f2d888b6-7c7a-46fd-9d0d-4928a2808e1e.png)
Back plate NOT shown in indented view

**Comment by jblacksberg** [comment]
![image](https://user-images.githubusercontent.com/105745183/218829441-10559823-7505-4771-96d8-1298d306d25e.png)
Backplate shown in flat view wtf

**Comment by jblacksberg** [comment]
omg it gets weirder. So one of the parts I tried to add but is hidden is called "TIMs Sub-Assembly". Same story as the "Backplate". 

I found where it put it! The app decided to add a 'folder' under an unrelated component called "(-) Neousys Nuvo Power Cable" _see first pic 



![Image](https://user-images.githubusercontent.com/105745183/218869069-c8a6828d-d27d-4fdb-832a-9864c455cf1d.png)


now if i open this part the TIMs sub assembly parts show up but in the wrong spot



![Image](https://user-images.githubusercontent.com/105745183/218869176-22132bc0-dc03-481b-96c1-759dbccc3839.png)
